sdk/src/manifest.rs

```diff
@@ -693,7 +693,8 @@ pub(crate) mod tests {
         let test_output = dir.path().join("wc_embed_test.jpg");
 
         //embed a claim generated from this manifest
-        let (signer, _) = get_temp_signer(&dir.path());
+        let cert_dir = fixture_path("certs");
```

Comment:
It may make sense to have this "certs" path defined as a constant in test_utils or something, at least for  all the unit tests. Or we could have another dedicated "certs_path() function that just returns this path, since it is used so often.